Tests/ELFParser/SymbolContents.lean

```diff
@@ -53,3 +58,14 @@ def getSymbolMap (symbolname : String) (elffile : RawELFFile) : IO Program := do
                       BitVec.ofNat 32 x')
                       insts
   return (BitVec.enumFrom (BitVec.ofNat 64 st_value) insts_bv)
+
+/- Get the instruction for symbol `symbolname` in the file `elffile`. -/
+def getSymbolInsts (symbolname : String) (elffile : RawELFFile) : IO (List (BitVec 32)) := do
```

Comment:
I *knew* we'd talk about this name! Isn't this function also used to grab data, not just instructions? So the `Insts` suffix is misleading.

Maybe generalize it to something like `getSymbolWords`, where a word is a 32-bit value?

Reply:
Good idea. Done!